automation/src/test/scala/org/broadinstitute/dsde/firecloud/pages/AuthenticatedPage.scala

```diff
@@ -15,10 +15,20 @@ abstract class AuthenticatedPage(implicit webDriver: WebDriver) extends FireClou
     ui.clickSignOut()
   }
 
+  def filter(searchText: String): Unit = {
+    ui.filter(searchText)
+  }
+
+  def getUser(): String = {
```

Comment:
Nit: IntelliJ likes to infer things about functions starting with `get`. I've been favoring `read` instead, i.e. `readUserEmail`.